GVRf/Framework/framework/src/main/AndroidManifest.xml

```diff
@@ -25,9 +25,12 @@
         android:minSdkVersion="19"
         android:targetSdkVersion="19" />
 
+    <uses-permission android:name="com.samsung.android.hmt.permission.READ_SETTINGS" />
```

Comment:
It could. Currently it would be in all backends that we have. I rather request the permission for Oculus since I can see that they are trying to detect the model. The permission will not be required by more recent GearVR services but there are versions out there that need it.. If you insist I can move it to the backends.